From e1e5629a050414f35d6c5c6ac8596211ab0953c2 Mon Sep 17 00:00:00 2001
From: Michael Lass <lass@mail.uni-paderborn.de>
Date: Fri, 10 Jun 2016 21:14:07 +0200
Subject: [PATCH 4/7] Remove boost detection

For Linux this only works on distributions using dpkg. The Arch Linux package
already makes sure that boost is installed.
---
 pri/utils.pri | 40 ----------------------------------------
 1 file changed, 40 deletions(-)

diff --git a/pri/utils.pri b/pri/utils.pri
index ae14ea8..eab99a0 100644
--- a/pri/utils.pri
+++ b/pri/utils.pri
@@ -17,46 +17,6 @@
 # $Date: 2013-01-12 07:45:08 +0100 (Sa, 12. Jan 2013) $
 # ********************************************************************/
 
-# boost_1_54_0 is buggy
-BOOSTS = 43 44 45 46 47 48 49 50 51 52 53 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99
-LATESTBOOST = 0
-for(boost, BOOSTS) {
-        exists(../src/lib/boost_1_$${boost}_0) {
-                LATESTBOOST = $$boost
-        }
-}
-
-contains(LATESTBOOST, 0) {
-    unix {
-        !macx {
-             BOOSTINFO = $$system(dpkg -s libboost-dev | grep 'Version')
-             BADVERSION = $$find(BOOSTINFO, 1\.54)
-             !isEmpty(BADVERSION) {
-                 message("Boost 1.54 has a bug in a function that Fritzing uses, so download or install some other version")
-                 error("Easiest to copy the boost library to .../src/lib/, so that you have .../src/lib/boost_1_xx_0")
-             }
-             isEmpty(BADVERSION) {
-                 BOOSTVERSION = $$find(BOOSTINFO, 1\...\.0)
-                 !isEmpty(BOOSTVERSION) {
-                     LATESTBOOST = installed
-                     message("using installed BOOST library")
-                 }
-             }
-        }
-    }
-}
-
-contains(LATESTBOOST, 0) {
-        message("Please download the boost library--you can find it at http://www.boost.org/")
-        message("Note that boost 1.54 has a bug in a function that Fritzing uses, so download some other version")
-        error("Copy the boost library to .../src/lib/, so that you have .../src/lib/boost_1_xx_0")
-}
-
-!contains(LATESTBOOST, installed) {
-    message("using boost from src/lib/boost_1_$${LATESTBOOST}_0")
-    INCLUDEPATH += src/lib/boost_1_$${LATESTBOOST}_0
-}
-
 HEADERS += \
 src/utils/abstractstatesbutton.h \
 src/utils/autoclosemessagebox.h \
-- 
2.8.3

